Fatal error: Uncaught Error: Call to private test::__construct() from invalid context in %s:%d
Stack trace:
#0 {main}
  thrown in %s on line %d